README.md

```diff
@@ -38,15 +38,14 @@ In the future we plan to integrate with several home automation systems. Below i
 
 |Integrations|Supported|Notes|
 |--|--:|--|
-|Home Assistant|Yes|See https://github.com/brianhealey/hacs_amplipi|
+|Home Assistant|Yes|See https://github.com/micro-nova/hacs_amplipi|
 |openHAB|Yes|See https://next.openhab.org/addons/bindings/amplipi/|
 |Alexa|No|See [#25](http://github.com/micro-nova/AmpliPi/issues/25)|
 |Google Assistant|No|See [#69](http://github.com/micro-nova/AmpliPi/issues/69)|
 
-## Quick Start Guide
+## Manual
 
-If you are one of the lucky few to have a pre-release version of the AmpliPi,
-see [docs/QUICK_START.md](docs/QUICK_START.md) to get started.
+Find the device manual [here](amplipi.com/manual)
```

Comment:
There is no pre-release AmpliPi, we've deleted QUICK_START.md in favor of the manual